modules/services/window-managers/hyprland.nix

```
@@ -40,7 +40,19 @@ in {
   ];
 
   options.wayland.windowManager.hyprland = {
-    enable = lib.mkEnableOption "Hyprland wayland compositor";
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Whether to enable configuration for Hyprland, a tiling wayland compositor that doesn't sacrifice on it's looks.
+
+        ::: {.note}
+        This module configures Hyprland and adds it to your users' `$PATH`, but does not make certain system-level changes.
```

```suggestion
        This module configures Hyprland and adds it to your user's `$PATH`, but does not make certain system-level changes.
```
Since options are per-user anyway